### PR TITLE
Update wall_thread.tpl

### PR DESCRIPTION
**Pull request by hoergen**
named an unnamed style definition to can address it via css

### CODE REVIEW

File: view/theme/frio/templates/wall_thread.tpl [hunk 1]
```diff
@@ -248,7 +248,7 @@ as the value of $top_child_total (this is done at the end of this file)
 		<div class="wall-item-content {{$item.type}}" id="wall-item-content-{{$item.id}}">
 			{{* insert some space if it's an top-level post *}}
 			{{if $item.thread_level==1}}
-			<div style="height:10px;">&nbsp;</div> <!-- use padding/margin instead-->
+			<div id="wall-spacer" style="height:10px;">&nbsp;</div> <!-- use padding/margin instead-->
```
Comment (tobiasd) [inline view/theme/frio/templates/wall_thread.tpl]: As `wall-spacer` now has a name; you could move the styling from the template (`height: 10px;`) to the main CSS file of frio. Your changes can then be appied in plusminus style as planned.

**Comment by hoergen** [comment]
Ok did the changes

File: view/theme/frio/templates/wall_thread.tpl [hunk 2]
```diff
@@ -248,7 +248,7 @@ as the value of $top_child_total (this is done at the end of this file)
 		<div class="wall-item-content {{$item.type}}" id="wall-item-content-{{$item.id}}">
 			{{* insert some space if it's an top-level post *}}
 			{{if $item.thread_level==1}}
-			<div style="height:10px;">&nbsp;</div> <!-- use padding/margin instead-->
+			<div id="wall-spacer">&nbsp;</div> <!-- use padding/margin instead-->
```
Comment (MrPetovan) [inline view/theme/frio/templates/wall_thread.tpl]: Can you be sure there will be only one `wall-spacer` in the whole page? I'd rather you used a class for that.

Comment (hoergen) [reply view/theme/frio/templates/wall_thread.tpl]: ok i changed it to a class

**Pull request merged**